conf/standalone.conf

```diff
@@ -424,6 +424,9 @@ authenticationProviders=
 # Enforce authorization
 authorizationEnabled=false
 
+# issuer url that points openid-configurations from a third party provider
+issuerUrl=
```

Comment:
What about adding some prefix here or some better name that gives a context ?
otherwise this name is not much meaningful if you pick it separated from the context.
This entry is useful only for the new Auth provider

Reply:
maybe OIDCIssuerUrl?

Reply:
what about `tokenAuthenticationOIDCIssuerUrl` ?